napari/components/_dims/qtdims.py

```diff
@@ -0,0 +1,232 @@
+from PyQt5.QtCore import Qt, pyqtSignal
+from PyQt5.QtWidgets import QWidget, QSlider, QGridLayout, QRadioButton
+from typing import Union
+
+from napari._qt.range_slider.range_slider import QVRangeSlider, QHRangeSlider
+
+from napari.components._dims.dims import DimsMode, Dims
+
+
+class QtDims(QWidget):
+    """
+        Qt View for Dims model.
+    """
+
+    _slider_height = 22
+
+    # Qt Signals for sending events to Qt thread
+    update_axis = pyqtSignal(int)
+    update_nbdims = pyqtSignal()
+
+
+    def __init__(self, dims: Dims, parent = None):
+        """
+        Constructor for Dims View
```

Comment:
Constructor docstrings do not go in `__init__`; they go in the class docstring instead.

Reply:
fixed